packages/core-components/src/components/SupportButton/SupportButton.test.tsx

```diff
@@ -15,31 +15,37 @@
  */
 
 import React from 'react';
+import { fireEvent, screen } from '@testing-library/react';
 import {
-  render,
-  act,
-  RenderResult,
-  waitFor,
-  fireEvent,
-  screen,
-} from '@testing-library/react';
-import { wrapInTestApp, renderInTestApp } from '@backstage/test-utils';
+  MockConfigApi,
+  renderInTestApp,
+  TestApiProvider,
+} from '@backstage/test-utils';
 import { SupportButton } from './SupportButton';
+import { configApiRef } from '@backstage/core-plugin-api';
+
+const configApi = {
+  getOptionalConfig: () =>
+    new MockConfigApi({
+      url: 'https://github.com',
+      items: [
+        {
+          title: 'Github',
+          icon: 'github',
+          links: [{ title: 'Github Issues', url: '/issues' }],
+        },
+      ],
+    }),
```

Comment:
You can use this directly rather than mocking the `getOptionalConfig` method. Just gotta wrap all of this data up into `app: { support: { ... } }`

Reply:
I did not know that, thanks! I updated it